R/rows-d.R

```diff
@@ -0,0 +1,214 @@
+#' Persisting data
+#'
+#' @description
+#' \lifecycle{experimental}
+#'
+#' These methods provide a framework for manipulating individual rows
+#' in existing tables, modeled after the SQL operations
+#' `INSERT`, `UPDATE` and `DELETE`.
+#' All operations expect existing and new data to be compatible.
+#'
+#' @param .data Target table object.
+#' @param .key Key columns, unquoted.
+#'   The default varies for the different operations:
+#'   - For `rows_insert()`: no keys.
+#'   - For `rows_update()`, `rows_patch()` and `rows_upsert()`: first column.
+#'   - For `rows_delete()`: all columns.
+#' @param ... New data.
```

Comment:
Should persist be part of the signature of the generic?

Reply:
`inplace = NULL`